tests/unit/plan_pretty_print.cpp

```diff
@@ -1077,3 +1077,57 @@ TYPED_TEST(PrintToJsonTest, Exists) {
             }
           })sep");
 }
+
+// Test for rollup apply operator
+TYPED_TEST(PrintToJsonTest, RollUpApply) {
+  Symbol x = this->GetSymbol("x");
+  Symbol e = this->GetSymbol("edge");
+  Symbol n = this->GetSymbol("node");
+  Symbol output = this->GetSymbol("output_symbol");
+  std::shared_ptr<LogicalOperator> list_collection_expand =
+      std::make_shared<Expand>(nullptr, x, n, e, memgraph::query::EdgeAtom::Direction::BOTH,
+                               std::vector<memgraph::storage::EdgeTypeId>{this->dba.NameToEdgeType("EdgeType1")}, false,
+                               memgraph::storage::View::OLD);
+  std::shared_ptr<LogicalOperator> list_collection_produce =
+      std::make_shared<Produce>(list_collection_expand, std::vector<NamedExpression *>{NEXPR("alias", IDENT("node"))});
+
+  std::shared_ptr<LogicalOperator> input_op = std::make_shared<ScanAll>(nullptr, x);
```

Comment:
You can use `auto` as you are using make::shared, so it will be shared_ptr for better readability